core/src/test/scala/com/vladkopanev/zio/saga/SagaSpec.scala

```diff
@@ -85,6 +88,39 @@ object SagaSpec
             } yield assert(log)(equalTo(Vector("flight canceled", "hotel canceled")))
           }
         ),
+        suite("Saga#zipWith")(
+          test("should successfully run two Sagas in sequence") {
+          for {
+            startTime <- currentTime(TimeUnit.MILLISECONDS).provide(Clock.live)
```

Comment:
Worth moving this current time pattern into a separate method in `SagaSpecUtil` so not to repeat yourself

Reply:
A more accurate version of the test wont be using similar pattern, so I think no need to extract it